content/work/medcurbside.rst

```diff
@@ -24,13 +24,15 @@ quotes:
       I defer to the team expertise now,
       which makes a better result.
     name: 'Rohit Puranik'
+    face: 'rohit-puranik.jpg'
     role: 'Founder'
     slug: 'goals'
   - text: |
       I liked the way OddBird presented themselves.
       **They were curious and asked detailed questions
       to really understand what was needed.**
     name: 'Rohit Puranik'
+    face: 'rohit-puranik.jpg'
```

Comment:
Did we get explicit permission to use these images of Rohit, Sara, and Furman?

Reply:
Nope - @SondraE do you want to reach out to them?